src/store/cartSlice/cartSlice.js

```diff
@@ -0,0 +1,50 @@
+import {createSlice, createAsyncThunk} from "@reduxjs/toolkit";
+import { getCart } from "api/Api";
+import {LOAD_STATUSES, CART} from "../constants";
+
+export const fetchCart = createAsyncThunk(`${CART}/getCategory`, async () => {
```

Comment:
```suggestion
export const fetchCart = createAsyncThunk(`${CART}/getCategory`, getCart)
```
у тебя сейчас твоя функция обертка ничего полезного не делает - просто получает из getCart данные и дальше передает. Поэтому можно сразу к getCart обращаться напрямую